Sources/MockableMacro/Requirements/Requirements.swift

```diff
@@ -20,7 +21,7 @@ struct Requirements {
 
     init(_ syntax: ProtocolDeclSyntax) throws {
         self.syntax = syntax
-
+        self.membersModifiers = syntax.modifiers.trimmed.excludingPrivateModifiers
```

Comment:
Let's move this down after the guard so it is next to the other property initializations